From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thiago Macieira <thiago.macieira@intel.com>
Date: Tue, 3 Jul 2018 16:38:38 -0700
Subject: [PATCH] Allow the groupadd -g command to fail

It fails for me consistently because I am using the same UID/GID inside
the container as outside and group ID 100 already exists in Clear.

Rebased for mock 2.4 update by Francisco Boni <boboniboni@gmail.com>.

Signed-off-by: Thiago Macieira <thiago.macieira@intel.com>
Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 mock/py/mockbuild/buildroot.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mock/py/mockbuild/buildroot.py b/mock/py/mockbuild/buildroot.py
index 1ed6fdd..f6156aa 100644
--- a/mock/py/mockbuild/buildroot.py
+++ b/mock/py/mockbuild/buildroot.py
@@ -425,7 +425,7 @@ class Buildroot(object):
 
         if self.chrootgid:
             self.doChroot(['/usr/sbin/groupadd', '-g', self.chrootgid, self.chrootgroup],
-                          shell=False, nosync=True)
+                          shell=False, raiseExc=False, nosync=True)
         self.doChroot(shlex.split(self.config['useradd']), shell=False, nosync=True)
         if not self.config['clean']:
             self.uid_manager.changeOwner(self.make_chroot_path(self.homedir))
